src/renderer/widgets/CreateWallet/lib/create-multisig-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { Step } from './types';
+
+export const createMultisigUtils = {
+  isStep,
+  isNotFirstStep,
+};
+
+/**
+ * Check if current step equal to the target step
+ * @param step  current step
+ * @param targetStep target step
+ * @return {boolean}
+ */
+function isStep(step: Step, targetStep: Step): boolean {
```

Comment:
We already have general `isStep` function, maybe use it?

Reply:
But the steps supported by `isStep` are different. Do you really want me to add my specific steps to the somewhat generic `Step` type? 

```
export const enum Step {
  NONE,
  INIT,
  CONFIRM,
  SIGN,
  SUBMIT,
  BASKET,
}
```

Reply:
Yes, we decided to do it in such way

Reply:
I made the `isStep` generic with types, so that anyone can check 2 steps. https://github.com/novasamatech/nova-spektr/pull/1800/commits/429570a7754570c9f6250966e9417b3ba8070a6c